pkg/sql/schemachanger/testdata/drop

```diff
@@ -143,10 +145,11 @@ commit transaction #1
 # begin PostCommitPhase
 begin transaction #2
 ## PostCommitPhase non-revertible stage 1 of 1 with 11 MutationType ops
+write *eventpb.DropSchema to event log for descriptor #57: DROP SCHEMA db.sc CASCADE
```

Comment:
Should we pull more structure out of this entry? How verbose would it be to print the whole entry?

Reply:
At some point we should. Right now, adding more detail doesn't seem useful. I'm happy to be convinced otherwise, though.